specs/WebResourceResponseReceived.md

```diff
@@ -0,0 +1,232 @@
+# Background
```

Comment:
Confused about the background ... The Requested event is raised when the JS makes a request? And this new event allows the hosting app to see the response before the JS does?

Reply:
Yes, the `WebResourceRequested` event is raised for requests from JS. But I'm not sure if this should be mentioned in the background for `WebResourceResponseReceived` as it is a different event. For `WebResourceResponseReceived`, the app can see the request/response as the browser process continues processing the response. The handler for this event does not block processing of the response.

Reply:
This is all good info to include in the API documentation:
 * The event is dispatched asynchronously after WebView2 receives the response and there is no ordering guarantee between WebView2 processing the response and the app host's event handler running.
 * A web request is any URI resolution the WebView2 performs. This includes declarative <img src=...> from HTML, or implicit favicon.ico look ups, or JavaScript in the document calling the fetch(...) API.

Reply:
Updated the document to include both.